openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -118,6 +118,54 @@ function aboutFeeds() {
 };
 
 
+var create_subject_carousel = function(subject_name) {
+    $.ajax({
+	dataType: "json",
+        url: '/carousels/subjects/' + subject_name,
+        type: "GET",
+        contentType: "text/html",
+        beforeSend: function(xhr) {
+            xhr.setRequestHeader("Content-Type", "application/json");
+            xhr.setRequestHeader("Accept", "application/json");
+        },
+        success: function(data) {
+	    // TODO: Filter `data` by available
+	    var page = new Subject(data, {pagesize: 6});
+	    
+	    function loadCoverCarousels (carousel, state) {
+		if (!page.coverCarousel) {
+		    page.coverCarousel = carousel;
+		    page.coverCarousel.size(page.getPageCount());
+		}
+		var index = carousel.first;
+		page.pos = (index-1)*6;
```

Comment:
Add whitespace around operators? (Applies everywhere.)

Also, the magic number 6 appears in a number of places. Turn into a constant?